cities/prague_pyvo.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+
+from calendar_data.data import parse_calendar_data
+from telegram_bot.bot import send_message
+
+
+def send_message_to_prague_channel():
+    events = parse_calendar_data("https://pyvo.cz/api/series/praha-pyvo.ics")
+    prague_chat_id = "-1001837942773"  # test chat
```

Comment:
Constants are generally UPPER_CASE and at the top of the file: https://peps.python.org/pep-0008/#constants